ArduCopter/GCS_Mavlink.h

```diff
@@ -65,11 +65,11 @@ class GCS_MAVLINK_Copter : public GCS_MAVLINK
     float vfr_hud_alt() const override;
 
     void send_pid_tuning() override;
-
     void send_winch_status() const override;
-
     void send_wind() const;
 
+    bool set_destination_target(const Location &dest_loc, bool use_yaw, float yaw_cd, bool use_yaw_rate, float yaw_rate_cds) const;
```

Comment:
I think it is confusing to have various routes into guided mode's set_destination_target.  So we already have two:

- Copter.h has a number of methods used by scripting (see Copter.h line 646)
- mode.h has ModeGuided::set_destination.

I'm not sure we should be adding more of these.

Reply:
Okay. I think I see what you mean